src/components/responsive.css

```diff
@@ -0,0 +1,144 @@
+/* responsive rules - starts here */
+
+/* mobile phone rules - starts here */
+@media (max-width: 390px) {
+    .menu-button-container {
+      display: flex;
+      z-index: 19;
+    }
+    .menu {
+    position: absolute;
+    top: 0;
+    left: 0;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    justify-content: flex-end;
+    align-items: center;
+    z-index: 12;
+    border: none;
+  }
+    #menu-toggle ~ .menu li {
+      height: 0;
+      margin: 0;
+      padding: 0;
+      transition: height 400ms cubic-bezier(0.24, 1, 0.33, 1);
+      z-index: 18;
+      cursor: pointer;
+    }
+    #menu-toggle:checked ~ .menu li {
+      border: 8px solid #fbf3f2;
+      height: 1.4rem;
+      padding: 7.8vh 0em 10.4vh 0em;
+      transition: height 400ms cubic-bezier(0.24, 1, 0.33, 1);
+      z-index: 18;
+    }
+    .menu > li {
+      display: flex;
+      justify-content: center;
+      margin: 0;
+      padding: 4.5em 0;
+      width: 100%;
+      color: #ff6666;
+      background-color: #fbf3f2;
+      cursor: pointer;
+    }
+    #logo {
+      margin: 10px 0 0 10px;
+      opacity: 0.8;
+      z-index:10;
+    }
+    
+    .logo-position {
+      display: flex;
+      flex-direction:row;
+      justify-content: flex-start;
+    }
+  }
+  
+  /* tablet rules - starts here */
+  @media (min-width: 391px) and (max-width: 1023px) {
```

Comment:
I've never seen min-width and max-width being mentioned in a media query before, looks more clear doing it this way!